PWGCF/MultiparticleCorrelations/Core/MuPa-DataMembers.h

```diff
@@ -90,35 +90,43 @@ struct EventByEventQuantities {
-  bool fBookQAParticleHistograms2D[eQAParticleHistograms2D_N] = {kTRUE};       // book or not this 2D histogram, see configurable cfBookQAParticleHistograms2D
+  bool fFillQAParticleHistograms2D = true;                                     // if false, all 2D histograms in this category are not filled. If true, the ones for which fBookQAParticleHistograms2D[...] is true, are filled
+  bool fBookQAParticleHistograms2D[eQAParticleHistograms2D_N] = {true};        // book or not this 2D histogram, see configurable cfBookQAParticleHistograms2D
   float fParticleHistogramsBins2D[eQAParticleHistograms2D_N][2][3] = {{{0.}}}; // [type - see enum][x,y][nBins,min,max]
   TString fParticleHistogramsName2D[eQAParticleHistograms2D_N] = {""};         // name of fQAParticleHistograms2D, determined programatically from other 1D names, to ease bookkeeping
 
   TList* fQAParticleEventList = NULL;                                                      //!<! base list to hold all QA particle event output object
   TH2F* fQAParticleEventHistograms2D[eQAParticleEventHistograms2D_N][2][2] = {{{NULL}}};   //! [ type - see enum eQAParticleEventHistograms2D ][reco,sim][before, after cuts]
   bool fFillQAParticleEventHistograms2D = true;                                            // if false, all 2D histograms in this category are not filled. If true, the ones for which fBookQAParticleEventHistograms2D[...] is true, are filled
-  bool fBookQAParticleEventHistograms2D[eQAParticleEventHistograms2D_N] = {kTRUE};         // book or not this 2D histogram, see configurable cfBookQAParticleEventHistograms2D
+  bool fBookQAParticleEventHistograms2D[eQAParticleEventHistograms2D_N] = {true};          // book or not this 2D histogram, see configurable cfBookQAParticleEventHistograms2D
   float fQAParticleEventHistogramsBins2D[eQAParticleEventHistograms2D_N][2][3] = {{{0.}}}; // [type - see enum][x,y][nBins,min,max]
   TString fQAParticleEventHistogramsName2D[eQAParticleEventHistograms2D_N] = {""};         // name of fQAParticleEventHistograms2D, determined programatically from other 1D names, to ease bookkeeping
   TProfile* fQAParticleEventProEbyE[2][2] = {{NULL}};                                      // helper profile to calculate <some-particle-property> event-by-event
                                                                                            // [reco, sim][before, after]. Type dimension is bin.
 
+  TList* fQACorrelationsVsList = NULL;                                                                  //!<! base list to hold all QA particle event output object
+  TH2F* fQACorrelationsVsHistograms2D[eQACorrelationsVsHistograms2D_N][gMaxHarmonic][2] = {{{{NULL}}}}; //! [ type - see enum eQACorrelationsVsHistograms2D ][reco,sim]. I do not have here support for [before, after], because I do not fill Q-vectors before cuts
```

Comment:
Shouldn't it be one less set of curl brackets?

Reply:
> Shouldn't it be one less set of curl brackets?

Yeap! It seems the compiler caught it before!

Reply:
I fixed it in the meantime! Thanks for the ultrafast review and approval - fully appreciated!